src/EnergyPlus/UtilityRoutines.hh

```diff
@@ -130,39 +130,45 @@ template <typename T, typename = std::enable_if_t<std::is_arithmetic_v<T>>> inli
 
 bool env_var_on(std::string const &env_var_str);
 
-using OptionalOutputFileRef = Optional<std::reference_wrapper<EnergyPlus::InputOutputFile>>;
+using OptionalOutputFileRef = std::optional<std::reference_wrapper<EnergyPlus::InputOutputFile>>;
```

Comment:
The horror. I have some dim recollection of why this turducken of an object might be necessary, but I've blocked it out and this is probably not the place for John Madden references. I am relatively certain that using `std::optional` here accomplishes the same thing, but I am happy to be corrected. As long as I never have to think about this again.